save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -134,27 +140,19 @@ class ProjectService(
      */
     fun getAllAsFluxByOrganizationName(organizationName: String) = getAllByOrganizationName(organizationName).let { Flux.fromIterable(it) }
 
-    /**
-     * @return project's without status
-     */
-    fun getNotDeletedProjects(): List<Project> {
-        val projects = projectRepository.findAll { root, _, cb ->
-            cb.notEqual(root.get<String>("status"), ProjectStatus.DELETED)
-        }
-        return projects
-    }
-
     /**
      * @param organizationName
      * @param authentication
+     * @param status
      * @return list of not deleted projects
      */
-    fun getNotDeletedProjectsByOrganizationName(
+    fun getProjectsByOrganizationNameAndStatus(
         organizationName: String,
         authentication: Authentication?,
+        status: ProjectStatus = ProjectStatus.CREATED
```

Comment:
Do we really need this default value? May be it's better to spend some time to add explicitly this status in code, and not it function definition? Function name `getProjectsByOrganizationNameAndStatus` have `status` in name, but in several places in code we don't provide `status`
Or even you can create additional function, something like that:
```kotlin
fun getProjectsByOrganizationNameAndCreatedStatus = getProjectsByOrganizationNameAndStatus(organizationName, authentication, ProjectStatus.CREATED)
```
Same as above could be said about other functions, where did you provide default value

Reply:
Now our projects/organizations can have 3 different statuses: ```CREATED```, ```DELETED```, ```BANNED```. 
Perhaps some other status will be added later.
For example, a search for several statuses may appear soon, so I don't think that we need to create a bunch of different functions. And in one function to search for everything, specifying the values of statuses. But it is usually the status ``CREATED`` that is used, so I made it the default value.

Reply:
fully agree with @kgevorkyan: `status` presents in method name, but missed when we call it. it little bit confuses

Reply:
Done